src/components/cards/PairPriceGraph/PairPriceGraph.vue

```diff
@@ -0,0 +1,235 @@
+<template>
+  <div ref="elementToAnimate" class="priceGraphCard">
```

Comment:
I think we should stick to kebab-case for css class names, as we have everywhere else:
```suggestion
  <div ref="elementToAnimate" class="price-graph-card">
```

Reply:
ok